Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -749,6 +753,8 @@ class BrowserViewController: UIViewController, BrowserViewControllerDelegate {
             }
         }
         
+        profile.prefs.setString(AppInfo.appVersion, forKey: LatestAppVersionProfileKey)
```

Comment:
Any reason why we do not use our `Preferences` for this?

Reply:
This is from legacy code from FF where they show a webpage showing whats new in actual version.
Removing this, it looks like it is not used anywhere else.

Reply:
Remove at https://github.com/brave/brave-ios/pull/4508/commits/429df1c4afce998ccc565b27120879a5afdfe87f